tests/unit/multi.tcl

```diff
@@ -306,15 +306,35 @@ start_server {tags {"multi"}} {
         r exec
     } {11}
 
-    test {MULTI / EXEC is propagated correctly (single write command)} {
+    test {MULTI / EXEC is not propagated (single write command)} {
         set repl [attach_to_replication_stream]
         r multi
         r set foo bar
         r exec
         assert_replication_stream $repl {
             {select *}
-            {multi}
             {set foo bar}
```

Comment:
here we don't validate that there no exec that messes things up
maybe add another command after the multi-exec and see that it's visible after the `set foo bar`